scripts/apps/authoring/authoring/authoring-custom-field.tsx

```diff
@@ -80,6 +80,7 @@ export class AuthoringCustomField extends React.PureComponent<IProps, IState> {
                             setValue={(value) => this.setValue(value)}
                             readOnly={!editable}
                             config={field.custom_field_config}
+                            template={this.props.template}
```

Comment:
can you remove template prop from here & `ITemplateEditorComponentProps` interface?